docs/skopeo.1.md

```diff
@@ -0,0 +1,115 @@
+% SKOPEO(1) Skopeo Man Pages
+% Jhon Honce
+% August 2016
+# NAME
+skopeo -- Various operations with container images images and container image registries
+# SYNOPSIS
+**skopeo [gobal options] command [command options]**
+# DESCRIPTION
+`skopeo` is a command line utility providing various operations with container images and container image registries. For example, it is able to inspect a repository on a Docker registry and fetch image. It fetches the repository's manifest and it is able to show you a `docker inspect`-like json output about a whole repository or a tag. This tool, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? which labels the image has?
+
+It also allows you to copy container images between various registries, possibly converting them as necessary, and to sign and verify images.
+## IMAGE NAMES
+Most commands refer to container images, using a `transport:details` format. The following formats are supported:
+
+  **atomic:**_namespace/stream:tag_
+  An image in the current project of the current default Atomic 
+  Registry. The current project and Atomic Registry instance are by 
+  default read from `~/.kube/config`, which is set e.g. using 
+  `(oc login)`.
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer
+  tarballs and signatures as individual files. This is a 
+  non-standardized format, primarily useful for debugging or 
+  noninvasive container inspection.
+
+  **docker://**_docker-reference_
+  An image in a registry implementing the "Docker Registry HTTP API V2".
+  By default, uses the authorization state in `~/.docker/config.json`, 
+  which is set e.g. using `(docker login)`.
+
+  **oci:**_path:tag_
+  An image _tag_ in a directory compliant with “Open Container Image 
+  Layout Specification” at _path_.
+
+# OPTIONS
+
+  _--debug_ enable debug output
+
+  _--username username_ for accessing  the registry
+
+  _--password password_ for accessing the registry
+
+  _--cert-path path_ Use certificates at _path_ (cert.pem, key.pem) to connect to the registry
+
+  _--tls-verify bool-value_ Verify certificates
+
+  _--help, -h_ Show help
+
+  _--version, -v_ print the version number
+
+# COMMANDS
+
+## skopeo copy _--sign-by=key-ID source-image destination-image_
```

Comment:
True, I couldn’t get that to work either;
```md
## skopeo copy [--sign-by=_key-ID_] _source-image destination-image_
```
parses the third `_` as content instead of markup for some reason.

Let’s leave that on one line using the compromise you are already using.